apis/sgv2-quarkus-common/src/test/java/io/stargate/sgv2/api/common/cql/builder/QueryBuilderValuesTest.java

```diff
@@ -225,4 +236,60 @@ public void shouldAllowCQLStatement() {
     Query query = new QueryBuilder().cql("SELECT * from system.local").build();
     assertThat(query.getCql()).isEqualTo("SELECT * from system.local");
   }
+
+  @Test
+  public void expressionTest() {
+    BuiltCondition condition = BuiltCondition.of("name", Predicate.EQ, TEST_NAME_VALUE);
+    BuiltCondition age = BuiltCondition.of("age", Predicate.EQ, TEST_AGE_VALUE);
+    BuiltCondition gender = BuiltCondition.of("gender", Predicate.CONTAINS, TEST_GENDER_VALUE);
+
+    Expression<BuiltCondition> expr =
+            And.of(Variable.of(condition), Or.of(Variable.of(age), Variable.of(gender)));
+    Query query =
+            new QueryBuilder().select().from("testKS", "testCollection").where(expr).limit(1).build();
+    System.out.println("ooo " + query.getCql());
```

Comment:
Please remove debug statements.

Reply:
Yes, initial purpose for this PR is to give Aaron an general idea. 
Will remove all the debug statements!